src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -492,6 +492,50 @@ export default class DefaultAudioVideoController
     );
   }
 
+  async replaceLocalVideo(): Promise<void> {
+    let videoStream: MediaStream | null = null;
+    try {
+      videoStream = await this.mediaStreamBroker.acquireVideoInputStream();
+    } catch (error) {
+      this.logger.info('could not acquire video stream from mediaStreamBroker');
```

Comment:
Should we log error here or throw Error like below?

Reply:
yeah, sounds good